recipe/meta.yaml

```diff
@@ -82,34 +80,37 @@ outputs:
     script: build-core.bat  # [win]
     build:
       skip: true  # [py<38]
+    build:
+      entry_points:
+        - ray = ray.scripts.scripts:main
     requirements:
       build:
         - {{ compiler('c') }}
         - {{ compiler('cxx') }}
         - patch          # [not win]
         - m2-patch       # [win]
-        - bazel ==4.2.2
+        - bazel =5.4.1
         - curl
-        - cython ==0.29.28
-        - git
+        - git  # [not win]
         - m2-make   # [win]
         - make      # [not win]
         - pkg-config  # [osx or (linux and aarch64)]
-        #- patchelf  # [linux]
-        - python  
+        - patchelf  # [linux]
+        - psutil
+        - python
         - sysroot_linux-64 2.17  # [linux and x86_64]
       host:
         - python
         - openjdk =11
         - pip
         - pkg-config  # [osx]
+        - cython >=0.29.32
         - setuptools
         - wheel
       run:
         - python
         - aiosignal
-        - attrs
-        - click >=7.0,<=8.0.4
+        - click >=7.0
         - colorama
         - filelock
         - frozenlist
```

Comment:
grpcio pinning should be python version dependent: https://github.com/ray-project/ray/blob/8a434b4ee7cd48e60fa1531315d39901fac5d79e/python/setup.py#L320

Reply:
I think we can ignore these pinnings in the anaconda ecosystem.